### PR TITLE
OpenAPI: zone distribution

### PR DESCRIPTION
**Pull request by jodosha**
Document `checkZoneDistribution` and `checkZoneRecordDistribution`.

Ref https://github.com/dnsimple/dnsimple-developer/pull/153

### CODE REVIEW

File: content/v2/openapi.yml [hunk 1]
```diff
@@ -23,7 +23,9 @@ servers:
     variables: {}
   - url: 'https://api.sandbox.dnsimple.com/v2'
     description: >-
-      DNSimple Sandbox API. For more information on the purpose and use of the Sandbox API, see our [testing documentation](https://developer.dnsimple.com/v2/#testing)
+      DNSimple Sandbox API. For more information on the purpose and use of the
+      Sandbox API, see our [testing
+      documentation](https://developer.dnsimple.com/v2/#testing)
```
Comment (aeden) [inline content/v2/openapi.yml]: Was the switch from 1 line to 3 lines intentional or the result of an automated tool?

Comment (jodosha) [reply content/v2/openapi.yml]: It was the export from the Swagger Editor.

File: content/v2/openapi.yml [hunk 2]
```diff
@@ -1216,6 +1218,38 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/ZoneDistribution'
+        '404':
+          $ref: '#/components/responses/404'
+        '504':
```
[the remark this answers is not in the document]
Comment (aeden) [reply content/v2/openapi.yml]: I only addressed the single issue you called out. I will fix this instance as well.

**Comment by aeden** [comment]
@weppos please take a look again when you have a moment.

**Pull request merged**